vite-plugins/jsBottomOfIndex.ts

```diff
@@ -0,0 +1,25 @@
+export const jsToBottomNoModule = () => {
+  return {
+    name: 'no-attribute',
+    transformIndexHtml(html: string) {
+      let replacedHtml = html.replace(`type="module" crossorigin`, '');
+      const matchResult = replacedHtml.match(
+        /<script[^>]*>(.*?)<\/script[^>]*>/,
+      );
+
+      if (matchResult) {
+        const scriptTag = matchResult[0];
+        console.log('\n SCRIPT TAG', scriptTag, '\n');
+        replacedHtml = replacedHtml.replace(scriptTag, '');
+        replacedHtml = replacedHtml.replace(
+          '<!-- # INSERT SCRIPT HERE -->',
```

Comment:
we could have done it without the marker, just search for `<body>` and replace with `<body> + <script>`